### DISCUSSION TITLE
Coupling two Fluent simulations with Python

### DISCUSSION THREAD

**Discussion by dmholler** [ansys/pyfluent#2969] [General]
Hi,
I'd like to couple two Fluent simulations with PyFluent. Has anyone tried to do this? I'm experienced in Fluent and Python separately and will be diving into PyFluent soon.

**Comment by dnwillia-work** [comment]
In principle I would say yes this is possible.  For instance, you can launch two sessions and couple them together using queries and BCs.  What sort of coupling did you have in mind?
